src/main/java/de/rwth/idsg/steve/config/SecurityConfiguration.java

```diff
@@ -100,23 +97,90 @@ public WebSecurityCustomizer webSecurityCustomizer() {
     @Bean
     public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
         final String prefix = CONFIG.getSpringManagerMapping();
-
         return http
-            .authorizeHttpRequests(
-                req -> req.antMatchers(prefix + "/**").hasRole("ADMIN")
-            )
+            .authorizeRequests(
+                 req -> req
+                    .antMatchers(prefix + "/home").hasAnyRole("USER", "ADMIN")
+                    .antMatchers(prefix + "/users/" + "**").hasAnyRole("USER", "ADMIN")
+                    .antMatchers(prefix + "/ocppTags/" + "**").hasAnyRole("USER", "ADMIN")
+                    .antMatchers(prefix + "/signout/" + "**").hasAnyRole("USER", "ADMIN")
+                    .antMatchers(prefix + "/noAccess/" + "**").hasAnyRole("USER", "ADMIN")
+                    .antMatchers(prefix + "/**").hasRole("ADMIN")
```

Comment:
Don't forget to manage the api route

Reply:
The api route is managed by the api key security filter chain. 
A similar approach for the api key or a combination of api key and user/password is possible, but needs to be discussed.

Reply:
Right. I forgot it 😅